backend/load-test/scripts/analyze-monitor.sh

```
@@ -0,0 +1,271 @@
+    state=$(echo "$state" | tr -d ' ')
+    count=$(echo "$count" | tr -d ' ')
+    [ -z "$state" ] || [ -z "$count" ] && continue
+    prev=${DB_ACT_PEAKS[$state]:-0}
+    if [ "$count" -gt "$prev" ] 2>/dev/null; then
+      DB_ACT_PEAKS[$state]=$count
+    fi
+  done < "$f"
+done
+
+# ----- Transaction commit/rollback rate -----
+TX_FIRST=$(ls ${MON_DIR}/tx_*.txt 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)